Docs/SDWebImage-5.0-Migration-guide.md

```diff
@@ -0,0 +1,94 @@
+## SDWebImage 5.0 Migration Guide
+
+SDWebImage 5.0 is the latest major release of SDWebImage, a top library for downloading and caching images.
+As a major release, following [Semantic Versioning](http://semver.org/) conventions, 5.0 introduces several API-breaking changes with its new architecture.
+
+.... RELEASE_SUMMARY ....
+
+This guide is provided in order to ease the transition of existing applications using SDWebImage 4.X to the latest APIs, as well as explain the design and structure of new and changed functionality.
+
+### Requirements: iOS 8, Mac OS X 10.10, watchOS 2, tvOS 9, Xcode 7.3
+
+SDWebImage 5.0 officially supports iOS 8 and later, Mac OS X 10.10 and later, watchOS 2 and later and tvOS 9 and later.
+It needs Xcode 7.3 or later to be able to build everything properly.
```

Comment:
...Actually, we already upgrate to use Xcode 8, or many feature like APNG, class property (For Swift user API) will fail. Please update this :)

Reply:
Done. I copy-pasted the Xcode 7.3 value until I could determine if an update is required. c38b731